src/Version.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace eLife\ApiClient;
+
+use PackageVersions\Versions;
+
+final class Version
+{
+    private static $version;
+
+    private function __construct()
+    {
+    }
+
+    public static function get() : string
+    {
+        if (empty(self::$version)) {
+            self::create();
+        }
+
+        return self::$version;
+    }
+
+    private static function create()
+    {
+        $originalVersion = Versions::getVersion('elife/api-client');
+        list($version) = explode('@', $originalVersion);
```

Comment:
what happens if the value is not what we expect? `list()` is silently throwing away part of it?